orm/model/ormtable/testdata/test_scenario.golden

```diff
@@ -1123,6 +1123,112 @@ ITERATOR 0100 -> 0101
       PK testpb.ExampleTable 9/0/ -> {"u32":9,"b":true}
   NEXT
   VALID false
+ITERATOR 0100 -> 0101
+  VALID true
+  KEY 010000047fffffffffffffff616263 10102203616263
+      PK testpb.ExampleTable 4/-1/abc -> {"u32":4,"u64":16,"str":"abc","bz":"abc","i64":-1}
+  KEY 010000047fffffffffffffff616263 10102203616263
+      PK testpb.ExampleTable 4/-1/abc -> {"u32":4,"u64":16,"str":"abc","bz":"abc","i64":-1}
+ORM UPDATE testpb.ExampleTable {"u32":4,"u64":16,"str":"abc","bz":"abc","i64":-1} -> {"u32":4,"u64":17,"str":"abc","bz":"abc","i64":-1}
```

Comment:
@fdymylja one question I have is about the ordering of calls to `Hooks`. You can see in the log here that the hook is called as soon as the write is batched, even though the actually SET and DELETE's are not applied later